components/test/src/main/java/org/trellisldp/test/InMemoryBinaryService.java

```diff
@@ -81,8 +81,8 @@ public String generateIdentifier() {
 
         private final byte[] data;
 
-        private InMemoryBinary(final byte[] data) {
-            this.data = data;
+        private InMemoryBinary(final InputStream data) throws IOException {
```

Comment:
I think this should be accompanied by a Javadoc note that the stream is consumed in this ctor.

Reply:
Good idea. I'll add that